coldfront/core/project/templates/project/project_request/savio/project_existing_pi.html

```diff
@@ -27,8 +27,13 @@ <h1>{{ PRIMARY_CLUSTER_NAME }}: Principal Investigator</h1><hr>
   <li>Allocation Period: {{ allocation_period.name }}</li>
 </ol>
 
-<p>Select an existing user to be a Principal Investigator of the project. You may search for the user in the selection field. If the desired user is not listed, you may skip this step and specify information for a new PI in the next step.</p>
-{% flag_enabled 'LRC_ONLY' as lrc_only %}
```

Comment:
Note: careful, this line was necessary to define the `lrc_only` boolean down below.